### ISSUE TITLE
Needs Updated For v3.4.2

### ISSUE THREAD

**Issue by bj47cf** [pbaity/rocketchat-dark-mode#42]
The administration page doesn't display correctly in dark mode in version 3.4.2. Drop down boxes for settings are almost impossible to read with dark mode enabled. The left side menu is no longer dark when dark mode is enabled.

The new version of rocketchat highlights each line of text in the chat channels that the mouse cursor is over. This causes it to display as white background on white text onhover which makes it unreadable.

Please & Thank You!

**Comment by araujorm** [comment]
Same here... for mouse-over the solution in https://github.com/pbaity/rocketchat-dark-mode/issues/41#issuecomment-655918851 seems to work (may need to clear cache aftewards), but the administration interface is impossible to use with dark mode (e.g. search text is invisible). Screenshot: 
![Selection_20200716(004)](https://user-images.githubusercontent.com/13053489/87669196-75c9b880-c765-11ea-9cac-8a5be3884e65.png)

**Comment by pbaity** [comment]
Just as a heads-up, it may be a little bit before I can get to this myself. Even once I have time, I need to install a local test instance of Rocket.Chat _without_ using the snap in order to work on this, and then work through all of the CSS changes in the newer version. 

I'm very happy to do this of course, but if anyone finds a fix for any of these problems, please open a PR. A PR doesn't need to address _all_ the problems at once either!

**Comment by pbaity** [comment]
It was really bugging me that I was letting this issues sit, so I started looking into this now anyway...

It looks like this may be a lot tougher than I thought. I opened PR #43 to fix the color of hovered messages, but looking into fixing the threads and admin panel I found a crap ton of classes that all look like `rcx-@d174ks1`, `rcx-@82jf7sk`, etc. These are generated classes, and they're pretty difficult to overwrite, and I doubt they have the same name from instance to instance. 

Looks like other people [have already been seeing this issue](https://github.com/RocketChat/Rocket.Chat/issues/17777) since the Rocket.Chat team is switching over to using [Fuselage](https://github.com/RocketChat/Rocket.Chat.Fuselage) for the UI.

To be honest, I'm not sure there's going to be a fix for this issue. I will keep looking for one though.
